addon/routes/oidc-authentication.js

```diff
@@ -31,6 +34,18 @@ export default class OIDCAuthenticationRoute extends Route {
     if (transition.from) {
       this.session.prohibitAuthentication(transition.from.name);
     }
+
+    // PKCE Verifier has to be set in session, because we redirect to keycloak and back
+    if (this.config.enablePkce) {
+      let pkceCodeVerifier = sessionStorage.getItem("pkceCodeVerifier");
+
+      if (!pkceCodeVerifier) {
+        pkceCodeVerifier = generateCodeVerifier(96);
+        sessionStorage.setItem("pkceCodeVerifier", pkceCodeVerifier);
+      }
+
+      this.pkceCodeVerifier = pkceCodeVerifier;
+    }
```

Comment:
For the `nextURL` we use the local storage (via session data) for the same reason. Let's keep it consistent and use the session as well. You can look at the implementation here: https://github.com/adfinis/ember-simple-auth-oidc/blob/80460c0019daab976d72912227d836e1d6adf3a1/addon/services/session.js#L67-L88

Reply:
Did not spot that, changed to use `session.data`